.github/workflows/wheels.yml

```diff
@@ -24,15 +24,44 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  macos:
-    uses: ./.github/workflows/wheels-macos.yml
-    with:
-      artifacts-name: "dist"
-
-  linux:
-    uses: ./.github/workflows/wheels-linux.yml
-    with:
-      artifacts-name: "dist"
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "macOS x86_64"
+            os: macos-latest
+            archs: x86_64
+          - name: "macOS arm64"
+            os: macos-latest
+            archs: arm64
+          - name: "manylinux2014 and musllinux x86_64"
+            os: ubuntu-latest
+            archs: x86_64
+          - name: "manylinux_2_28 x86_64"
+            os: ubuntu-latest
+            archs: x86_64
+            build: "*manylinux*"
+            manylinux: "manylinux_2_28"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.16.2
+        with:
+          output-dir: wheelhouse
+        env:
+          CIBW_ARCHS: ${{ matrix.archs }}
+          CIBW_BUILD: ${{ matrix.build }}
+          CIBW_CONFIG_SETTINGS: raqm=enable raqm=vendor fribidi=vendor
+          CIBW_MANYLINUX_PYPY_X86_64_IMAGE: ${{ matrix.manylinux }}
+          CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.manylinux }}
```

Comment:
> While arm64 wheels can be built on x86_64, they cannot be tested. The ability to test the arm64 wheels will be added in a future release of cibuildwheel, once Apple Silicon CI runners are widely available. To silence this warning, set `CIBW_TEST_SKIP: *-macosx_arm64`.

```suggestion
          CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.manylinux }}
          CIBW_TEST_SKIP: *-macosx_arm64
```

Reply:
Sure, I've added a commit for this. Testing, I found that it required quotes, as per https://cibuildwheel.readthedocs.io/en/stable/options/#test-skip

Reply:
I've created a PR to fix this upstream: https://github.com/pypa/cibuildwheel/pull/1665